srcpkgs/mvwm-git/patches/0001-Fix-cross-build.patch

```diff
@@ -0,0 +1,177 @@
+From 53b028752d6fdd96471be8cd4c3d1c8d490df2a5 Mon Sep 17 00:00:00 2001
+From: Nathan Owens <ndowens@artixlinux.org>
+Date: Fri, 25 Dec 2020 19:45:56 -0600
+Subject: [PATCH] Fix cross-build
+
+---
+ configure.ac | 77 ++++++++--------------------------------------------
+ 1 file changed, 12 insertions(+), 65 deletions(-)
+
+diff --git configure.ac configure.ac
+index 20e91fc..e7a544a 100644
+--- configure.ac
++++ configure.ac
+@@ -191,16 +191,6 @@ fi
+ AC_ISC_POSIX
+ AC_MINIX
+ 
+-# catch -Werror and similar options when running configure
+-AC_TRY_COMPILE([#include <stdio.h>],
+-[int i; static j; int *p; char *c;
+-  switch (*p = p = *c) { case 0: printf("%Q", c, p); }
+-  *c = &i; c = p;
+-  while (1 || (unsigned int)3 >= 0 || ((int)-1) == ((unsigned int)1));
+-  ], , AC_MSG_ERROR("
+-configure is not able to compile programs with warnings.  Please
+-remove all offending options like -Werror from the CFLAGS and
+-CPPFLAGS variables and run configure again."))
+ 
+ # check size of some types
+ ac_save_CFLAGS="$CFLAGS"
+@@ -540,22 +530,8 @@ if test ! x"$png_LIBS" = x; then
+   my_LIBS="$LIBS"
+   CPPFLAGS="$CPPFLAGS $png_CFLAGS"
+   LIBS="$LIBS $png_LIBS -lz -lm"
+-  AC_TRY_RUN([#include <png.h>
```

Comment:
I would try to minimize that patch we have. Let's say by just change `AC_TRY_RUN` to `AC_TRY_COMPILE`

Reply:
Doing that causes:
`configure.ac:113: error: possibly undefined macro: AC_DEFINE
      If this token and others are legitimate, please use m4_pattern_allow.`